README.md

```diff
@@ -59,6 +61,31 @@ Install specific files in Dom0:
 sudo qubesctl state.apply dotfiles.copy-dom0,dotfiles.copy-sh,dotfiles.copy-vim,dotfiles.copy-x11
 ```
 
+#### Pillar
+
+With Salt, the execution of each state can be opt out via the dotfiles pillar.
+You will need a top and a sls file in your pillar_roots (default to /srv/pillar,
+/srv/pillar/base, can be configured in /etc/salt/minion.d).
+
+Exemple: /srv/pillar/qusal/dotfiles.top to apply the dotfiles sls to all targets:
```

Comment:
`s/Exemple/Example/`

Reply:
Damn, I once lost a whole day on a file named licence instead of license >.< (the former is valid french as example is ...)